winpr/libwinpr/thread/thread.c

```diff
@@ -28,6 +28,14 @@
 
 #include <winpr/thread.h>
 
+#ifndef MIN
```

Comment:
not your fault, but we 13 definitions of `MIN` in FreeRDP and 5 for `MAX`...

Reply:
hmm, we already got a central definition for `freerdp` but lacking one for `winpr`